### PR TITLE
[PATCH v5] api: init: add new ODP_LOG_WARN log level

### PR DESCRIPTION
**Pull request by MatiasElo**

### CODE REVIEW

File: platform/linux-generic/include/odp/api/plat/debug_inlines.h [hunk 1]
```diff
@@ -54,6 +54,10 @@ extern odp_abort_func_t _odp_abort_fn;
 		 _ODP_LOG_FN(level, "%s:%d:%s(): " fmt, __FILE__, \
 		 __LINE__, __func__, ##__VA_ARGS__)
 
+#define _ODP_LOG_PREFIX(level, prefix, fmt, ...) \
```
Comment (TuomasTaipale) [inline platform/linux-generic/include/odp/api/plat/debug_inlines.h:57]: Having the prefix also in debug entries could make it easier to search relevant stuff from logs.

File: platform/linux-generic/arch/aarch64/odp_sysinfo_parse.c [hunk 2]
```diff
@@ -336,9 +336,8 @@ int _odp_cpuinfo_parser(FILE *file, system_info_t *sysinfo)
 			if (sysinfo->cpu_hz_max[id] == 0) {
 				uint64_t hz = sysinfo->default_cpu_hz_max;
 
-				_ODP_PRINT("WARN: cpu[%i] uses default max "
-					  "frequency of %" PRIu64 " Hz from "
-					  "config file\n", id, hz);
+				_ODP_WARN("CPU[%i] uses default max frequency of %" PRIu64 ""
+					  "Hz from config file\n", id, hz);
```
Comment (TuomasTaipale) [inline platform/linux-generic/arch/aarch64/odp_sysinfo_parse.c:340]: Whitespace before "Hz" so it's then same as before.

Comment (MatiasElo) [reply platform/linux-generic/arch/aarch64/odp_sysinfo_parse.c:340]: Will be fixed on V3.

File: platform/linux-generic/include/odp/api/plat/debug_inlines.h (hunk 1 above)
Comment (MatiasElo) [reply platform/linux-generic/include/odp/api/plat/debug_inlines.h:57]: Changed in V3.

**Review by TuomasTaipale — APPROVED** [review]

**Review by psavol — APPROVED** [review]

**Pull request merged**